crates/autopilot/src/run_loop.rs

```diff
@@ -246,28 +246,16 @@ impl RunLoop {
         let auction = self.remove_in_flight_orders(auction).await;
 
         // Mark all auction orders as `Ready` for competition
-        self.persistence.store_order_events(
-            auction.orders.iter().map(|o| OrderUid(o.uid.0)),
-            OrderEventLabel::Ready,
-        );
+        self.persistence
+            .store_order_events(auction.orders.iter().map(|o| o.uid), OrderEventLabel::Ready);
 
         // Collect valid solutions from all drivers
         let solutions = self.competition(&auction).await;
         observe::solutions(&solutions);
-
-        // Pick winners for execution
-        let winners = self.select_winners(&solutions);
-        if winners.is_empty() {
-            tracing::info!("no winners for auction");
```

Comment:
Would still be good to keep this log around IMO.

Reply:
Currently, if there are no valid solutions, we have a log for that.

If there is at least one valid solution, there is automatically at least one winner, so it can't happen that we have no winners in this case.